src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkCorrectUsingMethodsInspection.kt

```diff
@@ -43,19 +52,21 @@ class SmkCorrectUsingMethodsInspection: SnakemakeInspection() {
                 return
             }
 
+            val firstChildSection = st.firstChild
```

Comment:
Please try to avoid using low-level PSI API where possible, because it isn't stable. Use high-level PSI api provided by specific PSI element interfaces. Here you could use `st.sectionKeyword`

Reply:
Done